html-aam/index.html

```diff
@@ -16051,64 +16051,102 @@ <h2>Accessible Name and Description Computation</h2>
             </li>
-            <li>If the <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a> is still empty, then: use the control's `title` attribute.</li>
-            <li>Otherwise use the control's <a href="#att-placeholder">placeholder</a> value.</li>
             <li>If none of the above yield a usable text string there is no <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a>.</li>
           </ol>
         </section>
         <section>
-          <h4>`input type="button"`, `input type="submit"` and `input type="reset"` Accessible Name Computation</h4>
+          <h4>`input type="button"`, `input type="submit"` and `input type="reset"` Elements Accessible Name Computation</h4>
           <ol>
             <li>
-              If the control has an <a data-cite="wai-aria-1.2/#aria-label">`aria-label`</a> or an <a data-cite="wai-aria-1.2/#aria-labelledby">`aria-labelledby`</a> attribute the
+              If the control has an <a data-cite="wai-aria-1.2/#aria-label">`aria-label`</a> 
+              or an <a data-cite="wai-aria-1.2/#aria-labelledby">`aria-labelledby`</a> attribute the
               <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a> is to be calculated using the algorithm defined in
               <a href="" class="accname">Accessible Name and Description: Computation and API Mappings</a>.
             </li>
             <li>
-              Otherwise use the associated `label` element(s) <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a>(s) - if more than one `label` is associated; concatenate by DOM order,
-              delimited by spaces.
+              If the <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a> is still empty: 
+              use the <a data-cite="accname-1.2/#mapping_additional_nd_te">text equivalent computation</a> of the associated `label` element's subtree 
+              - if more than one `label` is associated; concatenate their subtrees by DOM order, delimited by spaces.
+              <p>
+                If the control is encapsulated by its `label` element, and the control has an author specified `value` or the lack of a `value`
+                has produced an <a href="https://infra.spec.whatwg.org/#implementation-defined">implementation defined</a> string to render, 
+                then exclude either from the control's computed <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a>.
+              </p>
+            </li>
+            <li>
+              If the <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a> is still empty: 
+              use the value of the control's `value` attribute.</li>
+            <li>
+              For `input type=submit` and `type=reset`: 
+              if the <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a> is still empty, and the `value` attribute is unspecified use the
```

Comment:
tiny nit, maybe add comma for readability and consistency

```suggestion
              if the <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a> is still empty, and the `value` attribute is unspecified, use the
```